app/controllers/landing_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LandingController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
since `LandingController` inherits from `ApplicationController`, we don't actually need to lift all those instance variables and what not being set in in the parent, they're happening here behind the scenes